package.json

```diff
@@ -20,39 +20,38 @@
   },
   "dependencies": {
     "body-parser": "^1.19.0",
-    "chalk": "^2.4.2",
-    "express": "^4.16.4",
-    "express-gateway": "^1.16.3",
-    "express-openapi": "^4.6.4",
-    "express-winston": "^3.1.0",
-    "fabric-client": "^1.4.3",
-    "fabric-network": "^1.4.1",
-    "fabric-ca-client": "^1.4.3",
-    "js-yaml": "^3.13.1",
-    "lodash": "^4.17.15",
-    "openapi-request-validator": "^3.8.3",
-    "openapi-response-validator": "^3.8.2",
-    "swagger-parser": "^8.0.0",
-    "swagger-ui-express": "^4.0.7",
-    "uuid": "^3.3.2",
-    "winston": "^3.2.1"
+    "chalk": "^4.1.0",
+    "express": "^4.17.1",
+    "express-gateway": "^1.16.10",
+    "express-openapi": "^7.3.0",
+    "express-winston": "^4.0.5",
+    "fabric-network": "^2.2.11",
+    "fabric-ca-client": "^2.2.11",
+    "js-yaml": "^4.0.0",
+    "lodash": "^4.17.20",
+    "openapi-request-validator": "^7.3.0",
+    "openapi-response-validator": "^7.2.3",
+    "@apidevtools/swagger-parser": "^10.0.2",
+    "swagger-ui-express": "^4.1.6",
+    "uuid": "^8.3.2",
+    "winston": "^3.3.3"
   },
   "devDependencies": {
-    "@types/express": "^4.16.1",
-    "@types/express-winston": "^3.0.1",
+    "@types/express-winston": "^4.0.0",
+    "@types/express": "^4.17.11",
     "@types/jest": "^27.0.3",
-    "@types/js-yaml": "^3.12.1",
-    "@types/lodash": "^4.14.123",
+    "@types/js-yaml": "^4.0.0",
+    "@types/lodash": "^4.14.168",
     "@types/node": "^12.12.14",
-    "@types/swagger-jsdoc": "0.0.2",
-    "@types/swagger-ui-express": "^3.0.1",
-    "@types/uuid": "^3.4.4",
-    "@types/validator": "^10.11.1",
+    "@types/swagger-jsdoc": "6.0.0",
+    "@types/swagger-ui-express": "^4.1.2",
+    "@types/uuid": "^8.3.0",
+    "@types/validator": "^13.1.3",
     "jest": "^27.4.3",
-    "nyc": "^15.0.0",
+    "nyc": "^15.1.0",
     "ts-jest": "^27.1.1",
-    "ts-node": "^8.5.4",
-    "tslint": "^5.20.1",
-    "typescript": "^3.4.5"
+    "ts-node": "^9.1.1",
+    "tslint": "^6.1.3",
```

Comment:
maybe we should use eslint here aswell? 😃

Reply:
sure. it is already in the next PR
https://github.com/arvenis/angus-router/pull/17
it is still in draft.